src/Modal/Modal.test.jsx

```diff
@@ -127,6 +127,25 @@ describe('<Modal />', () => {
       expect(modalFooter.find('button')).toHaveLength(1);
       expect(wrapper.find('button')).toHaveLength(1);
     });
+
+    it('renders with IE11-specific styling when IE11 is detected', () => {
+      const { MSInputMethodContext } = global;
+      const { documentMode } = global.document;
+
+      // mimic IE11
+      global.MSInputMethodContext = true;
+      global.document.documentMode = true;
+      wrapper = mount(<Modal {...defaultProps} />);
+      let modal = wrapper.find('.modal');
+      expect(modal.hasClass('is-ie11')).toEqual(true);
+
+      // return globals back to not IE11
+      global.MSInputMethodContext = MSInputMethodContext;
```

Comment:
Nit: this feels likes two tests jammed into one

1. has `is-ie11` class when browser is IE11
2. does not have `is-ie11` class when browser is not IE11